### PR TITLE
JENA-1913: Multiple --file on the Fuseki command line

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/FusekiCmd.java [hunk 1]
```diff
@@ -246,21 +247,26 @@ protected void processModulesAndArgs() {
...
+                List<String> filenames = getValues(argFile);
                 // Directly populate the dataset.
                 cmdLineConfig.reset();
                 cmdLineConfig.dsg = DatasetGraphFactory.createTxnMem();
-                Lang language = RDFLanguages.filenameToLang(filename);
-                if ( language == null )
-                    throw new CmdException("Can't guess language for file: " + filename);
-                Txn.executeWrite(cmdLineConfig.dsg, ()->RDFDataMgr.read(cmdLineConfig.dsg, filename));
+                String desc = filenames.size() == 1
+                    ? "file: "+filenames.get(1)
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/FusekiCmd.java:255]: .get(0)? (Spent the day looking at JS and Python, and on mobile, so I might be confusing langs... if so ignore please)

File: jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/cmds/FusekiMain.java [hunk 2]
```diff
@@ -282,21 +282,26 @@ protected void processModulesAndArgs() {
...
-            serverConfig.dsg = DatasetGraphFactory.createTxnMem();
-
-            // INITIAL DATA.
-            Lang language = RDFLanguages.filenameToLang(filename);
-            if ( language == null )
-                throw new CmdException("Can't guess language for file: " + filename);
-            Txn.executeWrite(serverConfig.dsg,  ()->RDFDataMgr.read(serverConfig.dsg, filename));
+            List<String> filenames = getValues(argFile);
+            String desc = filenames.size() == 1
+                ? "file: "+filenames.get(1)
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/cmds/FusekiMain.java:287]: Isn't the parameter the index of the element? Or is it the number of elements to pop/shift?

File: jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/FusekiCmd.java (hunk 1 above)
Comment (afs) [reply jena-fuseki2/jena-fuseki-webapp/src/main/java/org/apache/jena/fuseki/cmd/FusekiCmd.java]: Arrg! `.get(0)` both times (quick fix applied).

The messages need refining as well. Logging works different in the two different server startup sequences ("legacy" - sigh).

**Comment by afs** [comment]
Better information, same style output in each server. Better handling of bad syntax.

**Pull request merged**